src/components/Restaurants/WriteReviewPage.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import StarRatingInput from './StarRatingInput'; 
+import './WriteReviewPage.css';
+
+function WriteReviewPage() {
+    const [review, setReview] = useState("");  
+    const [submitting, setSubmitting] = useState(false);
+    const [submitted, setSubmitted] = useState(false);
+    const [rating, setRating] = useState(0);
+    const [selectedTags, setSelectedTags] = useState([]);
+
+    const handleInputChange = (event) => {
+        setReview(event.target.value);
+    };
+
+    const handleSubmit = (event) => {
+        event.preventDefault(); 
+        if (review.trim() === "" || rating === 0) {
+            alert("Please enter a review and select a rating before submitting.");
+            return;
+        }
+
+        setSubmitting(true);
+
+        setTimeout(() => {
+            console.log('Review Submitted:', review); 
+            setSubmitting(false);
+            setSubmitted(true);
+            setReview(""); 
+            setRating(0);
+            setSelectedTags([]);
+        }, 1000); 
+    };
+
+    const toggleTag = (tag) => {
+        setSelectedTags((prevTags) => 
+            prevTags.includes(tag) ? prevTags.filter(t => t !== tag) : [...prevTags, tag]
+        );
+    };
+
+    return (
+        <div>
+            <header className="header">
+                <div className="logo">
+                    <FaYelp size="30" style={{ color: 'white' }} /> YYYYYelppppppp
+                </div>
+                <div>
+                    <button className="navLink">Log In</button>
+                    <button className="navLink-button">Sign Up</button>
+                </div>
+            </header>
+            <div className="write-review-page">
+                <h2>Popeyes Louisiana Kitchen</h2>
+                {submitted ? (
+                    <Link to="/detail/write-review/success" className="submit-success-link">
+                        Review submitted! Click here to see your live review.
+                    </Link>
+                ) : (
+                    <form onSubmit={handleSubmit}>
+                        <StarRatingInput setRating={setRating} />
+                        <textarea 
+                            value={review}
```

Comment:
The textarea should be inside the box.
<img width="572" alt="Capture" src="https://github.com/HHHHHeloise/Yepyep/assets/120807105/7b3bff99-4ff4-4a39-bd73-565b9e9eedb9">